js/gated-content/src/filters/index.js

```diff
@@ -15,27 +15,23 @@ const filters = [
     },
   },
   {
-    name: 'schedule',
```

Comment:
Did we delete the needed code accidentally?

Reply:
it was unused code